src/main/java/common/helpers/BrowserHelper.java

```diff
@@ -72,4 +68,16 @@ public static void returnPreviousPage() {
         JavascriptExecutor js = (JavascriptExecutor) driver;
         js.executeScript("window.history.go(-1)");
     }
+
+    public static String getTitle(){
+        return getDriver().getTitle();
+    }
+
+    public static void handleAlert(boolean action) {
```

Comment:
action is not a good name for a boolean. action value is true or false doesn't make sense
you should you an enum for this and action could be either Accept or Dismiss

Reply:
thanks for letting me know, i removed the handleAlert method and replaced it with the alternate method with two methods: acceptAlert and allowAlert